sklearn/ensemble/_hist_gradient_boosting/tests/test_binning.py

```diff
@@ -20,35 +20,38 @@ def _find_binning_thresholds(data, max_bins=256, subsample=int(2e5),
                              random_state=None):
     # Just a redef to avoid having to pass arguments all the time (as the
     # function is private we don't use default values for parameters)
-    return _find_binning_thresholds_orig(data, max_bins, subsample,
-                                         random_state)
+    binning_thresholds = _find_binning_thresholds_orig(
+        data, max_bins, subsample, random_state)
+    return binning_thresholds
 
 
 def test_find_binning_thresholds_regular_data():
     data = np.linspace(0, 10, 1001).reshape(-1, 1)
-    bin_thresholds = _find_binning_thresholds(data, max_bins=10)
-    assert_allclose(bin_thresholds[0], [1, 2, 3, 4, 5, 6, 7, 8, 9])
+    bin_thresholds = _find_binning_thresholds(data, max_bins=11)
+    assert_allclose(bin_thresholds[0], [np.nan, 1, 2, 3, 4, 5, 6, 7, 8, 9])
```

Comment:
Note for reviewers: I had to adjust a lot of the tests in this file since the first bin is always allocated for missing values now.